buildscript/build.go

```diff
@@ -166,6 +166,10 @@ func runTests(ctx *task.Context, pkgs []string, testType string) error {
 			env = append(env, testtype.ReplSetTestType+"=true")
 		}
 
+		if ctx.Get("race") == "true" {
+			args = append(args, "-race")
```

Comment:
I'm seeing a [warning](https://parsley.mongodb.com/evergreen/mongo_tools_rhel88_race_integration_4.0_patch_5d25238d53ea5d8a304e954459f659f39a160156_669e9b1105cd0400064099d7_24_07_22_17_46_59/0/task?bookmarks=0,773,7529) about unused argument "race". It seems that it's not working.

Reply:
This should be fixedd.